Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -1546,6 +1546,14 @@ script:
       name: offset
       required: false
       secret: false
+    - default: false
+      description: 'System parameters. overrides the query, limit and offset arguments.
```

Comment:
it should not override the other query params IMO. Just add more params.

Reply:
If we want it not. to override at all and just add, we need to make it key=val, and then check waht happens if one of these keys are, e.g `sysparm_limit`.
I think this is not the preferred way.

Reply:
I think that usually users will want to add additional params, they will not like to add the query and limit themselves because it's already there. @Arsenikr WDYT?